src/components/Layout.js

```diff
@@ -107,20 +103,9 @@ export default function Layout({
               flexDirection: 'column',
             }}
           >
-            <SvgIcon
-              onClick={handleZoomIn}
-              component={ZoomIn}
-              inheritViewBox
-              sx={{ mb: '10px', height: 52, width: 52 }}
-            />
-            <SvgIcon
-              onClick={handleZoomOut}
-              component={ZoomOut}
-              inheritViewBox
-              sx={{ height: 52, width: 52 }}
-            />
+            <ZoomInOutButton moveEnd={zoomChanged} />
           </Box>
-          <App />
+          <App updateZoomChanged={updateZoomChanged} />
```

Comment:
This is a pretty tight coupling by changing the interface of this fundamental component,  is there any other light-weight way to reach the goal? For example, let's the zoom-in-out component register to listener the event directly? 
```
ZoomComponent = () => {
  useEffect(() => {
      ...get the map
      map.on(MOVE_END_EVENT, () => {
          ...check the map status
   })

```

Reply:
Ok, I will changed it, thank you.